tests/test_load_from_msp.py

```diff
@@ -0,0 +1,14 @@
+import os
+from matchms.importing import load_from_msp
+
+
+def test_load_from_msp():
+
+    module_root = os.path.join(os.path.dirname(__file__), "..")
+    spectrums_file = os.path.join(module_root, "tests", "MoNA-export-GC-MS.msp")
+
+    spectrums = load_from_msp(spectrums_file)
+    print(spectrums)
```

Comment:
Instead of printing it would be nice to also check some of the results with ``assert``.
```
assert spectrums[0].get("inchikey") == "whateveritis"
etc.
```

Reply:
Thank you, I will add it.

Reply:
Perfect, thanks.